### ISSUE TITLE
Review handling of Helgen Reborn new buildings

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#227]
Discord report

> would you please have the mod treat all of the new houses and businesses of Helgen Reborn as not to be scanned?  I have to keep remembering to pause it and I'm sure I can't be the only one.

**Comment by SteveTownsend** [comment]
This change looks analogous to what we did for the Helgen Tower to make it a Player House - need to treat all of the vendors houses as part of an artificial Location to trigger Location-based autoloot prevention. Seems like treating the new Helgen as a Town would be correct (rather than Settlement or City)

**Comment by SteveTownsend** [comment]
Fix applied and tested data load for v3.7.0.0.
```
2020-09-26 10:18:03.331     info  51300 Found exact match 0xa7031f4c for Helgen Reborn.esp:0x031f4c
2020-09-26 10:18:03.331     info  51300 Cell Potent Potables/0xa7031f4c treated as Town
2020-09-26 10:18:03.331     info  51300 Found exact match 0xa7031eac for Helgen Reborn.esp:0x031eac
2020-09-26 10:18:03.331     info  51300 Cell Helgen Wholesale/0xa7031eac treated as Town
2020-09-26 10:18:03.331     info  51300 Found exact match 0xa707fd96 for Helgen Reborn.esp:0x07fd96
2020-09-26 10:18:03.331     info  51300 Cell Empty House/0xa707fd96 treated as Town
2020-09-26 10:18:03.331     info  51300 Found exact match 0xa70320dd for Helgen Reborn.esp:0x0320dd
2020-09-26 10:18:03.332     info  51300 Cell Cienna's House/0xa70320dd treated as Town
2020-09-26 10:18:03.332     info  51300 Found exact match 0xa7031d3b for Helgen Reborn.esp:0x031d3b
2020-09-26 10:18:03.332     info  51300 Cell Helgen Inn/0xa7031d3b treated as Town
2020-09-26 10:18:03.332     info  51300 Found exact match 0xa71226d7 for Helgen Reborn.esp:0x1226d7
2020-09-26 10:18:03.332     info  51300 Cell Froki and Haming's House/0xa71226d7 treated as Town
2020-09-26 10:18:03.332     info  51300 Found exact match 0xa707fe11 for Helgen Reborn.esp:0x07fe11
2020-09-26 10:18:03.332     info  51300 Cell Hella & Dunroe's House/0xa707fe11 treated as Town
2020-09-26 10:18:03.332     info  51300 Found exact match 0xa707f1d8 for Helgen Reborn.esp:0x07f1d8
2020-09-26 10:18:03.332     info  51300 Cell Marcus' House/0xa707f1d8 treated as Town
2020-09-26 10:18:03.332     info  51300 Found exact match 0xa7031731 for Helgen Reborn.esp:0x031731
2020-09-26 10:18:03.332     info  51300 Cell Tall Mast Tavern/0xa7031731 treated as Town
2020-09-26 10:18:03.332     info  51300 Found exact match 0xa707fe8c for Helgen Reborn.esp:0x07fe8c
2020-09-26 10:18:03.332     info  51300 Cell House/0xa707fe8c treated as Town
2020-09-26 10:18:03.332     info  51300 Found exact match 0xa7060154 for Helgen Reborn.esp:0x060154
2020-09-26 10:18:03.332     info  51300 Cell Reinhardt's House/0xa7060154 treated as Town
2020-09-26 10:18:03.332     info  51300 Found exact match 0xa707fd1b for Helgen Reborn.esp:0x07fd1b
2020-09-26 10:18:03.332     info  51300 Cell Meen-Lei's House/0xa707fd1b treated as Town
2020-09-26 10:18:03.332     info  51300 Found exact match 0xa7031f41 for Helgen Reborn.esp:0x031f41
2020-09-26 10:18:03.332     info  51300 Cell The Arcane Ambry/0xa7031f41 treated as Town
2020-09-26 10:18:03.332     info  51300 Found exact match 0xa7032059 for Helgen Reborn.esp:0x032059
2020-09-26 10:18:03.332     info  51300 Cell Hammer and Anvil/0xa7032059 treated as Town
2020-09-26 10:18:03.332    trace  51300 Added 14 CELLs as Town, expected 14
```
